game.go

```diff
@@ -14,26 +14,52 @@ const (
 type Game struct {
     board [][]uint8
     currentMove Player
+    gameBoardSize int //added board size variable for testing out of scope moves
 }
 
 func NewGame(boardSize int) Game {
     board := make([][]uint8, boardSize)
     for i := range board {
         board[i] = make([]uint8, boardSize)
     }
-    newGame := Game{board: board, currentMove: DARK}
+    board = boardStars(board, boardSize)
+    newGame := Game{board: board, currentMove: DARK, gameBoardSize: boardSize}
     return newGame
 }
 
+func boardStars(board [][] uint8, boardSize int) [][] uint8 {
```

Comment:
I think this function should be like the `placePiece()` function, where it is an extension function on the `Game` model. Then it wouldn't need any extra arguments besides the game object itself.
Also, I think it should be `addBoardStars()`

Reply:
Also a really good point. Now called separately in main(), acts as a method for a game struct, and works with no arguments.